CHANGELOG.md

```diff
@@ -54,6 +55,7 @@
 - `WorkerConnector` no longer destroys itself in `Dispose`. [#1365](https://github.com/spatialos/gdk-for-unity/pull/1365)
 - `MultiThreadedSpatialOSConnectionHandler` and `SpatialOSConnectionHandlerBuilder.SetThreadingMode` have been removed. [#1367](https://github.com/spatialos/gdk-for-unity/pull/1367)
 - Command request IDs are now typed as `CommandRequestID` instead of `long`. [#1372](https://github.com/spatialos/gdk-for-unity/pull/1372)
+- C# enums will be generated to start from 0 and shifted to schema values on serialization and shifted back to C# values on deserialization.  A warning will be generated when enums defined in schema does not start from 0. [#1412](https://github.com/spatialos/gdk-for-unity/pull/1412)
```

Comment:
You should create a `### Breaking Changes` section above `### Added` and move this there.

On top of moving it to breaking changes:

```suggestion
- All generated C# enums will now start from 0, being shifted to schema values on serialization and shifted back to C# values on deserialization. [#1412](https://github.com/spatialos/gdk-for-unity/pull/1412)
	- A warning will be issued when enums defined in schema does not start from 0. 
```

Reply:
Ah this should be under Unreleased > Breaking Changes